From b45cc49d0000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Wed, 21 May 2026 00:00:00 +0000
Subject: [PATCH] Update elements/include/linux.yml to v7.0.9-0

Updates elements/include/linux.yml to v7.0.9-0 (9021cc1)
Previous version was v7.0.1-0 (3cb1fb7)
---
 elements/include/linux.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/include/linux.yml b/elements/include/linux.yml
index 0d70a2471d..9ae31219e2 100644
--- a/elements/include/linux.yml
+++ b/elements/include/linux.yml
@@ -4,6 +4,6 @@ sources:
   track: v*
   exclude:
   - '*-rc*'
-  ref: v7.0.1-0-g3cb1fb7a56d2fd8011f5282bc170c0d23dc1f4b5
+  ref: v7.0.9-0-g9021cc14f7d98b4a1d2c932f52c5343d4d0f6b92
 - kind: patch_queue
   path: patches/linux
-- 
GitLab

